### PR TITLE
small fixes to Amazon Q tree nodes in toolkit view

### PR DESCRIPTION
**Pull request by hayemaxi**
- Update string for Enable Amazon Q "Enable Amazon Q (Preview)" -> "Enable"
285990599-5309c3c8-47a2-4085-bbcf-48e2bac2254c

- "Switch To Amazon Q" tree node button does not always work for all clients. It is a silent
failure.
  - Solution: Use an alternative generated command to open the chat view.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/amazonq/explorer/amazonQChildrenNodes.ts [hunk 1]
```diff
@@ -55,7 +51,7 @@ export const switchToAmazonQNode = () =>
 export const enableAmazonQNode = () =>
     // Simply trigger re-auth to obtain the proper scopes- same functionality as if requested in the chat window.
     reconnect.build(placeholder, cwTreeNodeSource).asTreeNode({
-        label: localize('AWS.amazonq.enable', 'Enable Amazon Q (Preview)'),
+        label: localize('AWS.amazonq.enable', 'Enable'),
         iconPath: getIcon('vscode-debug-start'),
```
Comment (nkomonen-amazon) [inline src/amazonq/explorer/amazonQChildrenNodes.ts:55]: Just verifying, are we sticking with the start button?

**Review by nkomonen-amazon — APPROVED** [review]

File: src/amazonq/explorer/amazonQChildrenNodes.ts (hunk 1 above)
Comment (hayemaxi) [reply src/amazonq/explorer/amazonQChildrenNodes.ts:55]: Yup, verified the icon.

File: src/auth/ui/vue/show.ts [hunk 2]
```diff
@@ -843,3 +843,11 @@ export async function emitWebviewClosed(authWebview: ClassToInterfaceType<AuthWe
         return result
     }
 }
+
+/**
+ * Forces focus to Amazon Q panel - USE THIS SPARINGLY (don't betray customer trust by hijacking the IDE)
+ * Used on first load, and any time we want to directly populate chat.
+ */
+export async function focusAmazonQPanel(): Promise<void> {
```
Comment (nkomonen-amazon) [inline src/auth/ui/vue/show.ts:851]: Did this need to be moved here, was it to fix a circular dependency issue?

Comment (hayemaxi) [reply src/auth/ui/vue/show.ts:851]: Yes, something had to be moved from one of the 2 files, of which both things were reference across multiple other files. This made sense IMO but was arbitrary.

**Review by andrewyuq — APPROVED** [review]

**Pull request merged**